zhaquirks/sonoff/basiczbr3.py

```
@@ -0,0 +1,46 @@
+"""Device handler for Sonoff BASICZBR3."""
+
+from zigpy.quirks.v2 import QuirkBuilder
+
+from zhaquirks.const import (
+    CLUSTER_ID,
+    COMMAND,
+    ENDPOINT_ID,
+    LONG_PRESS,
+    PARAMS,
+    SHORT_PRESS,
+)
+
+# Define action types for the BASICZBR3
+POWER_ON = "power_on"
+FLASH = "flash"
+
+(
+    QuirkBuilder("SONOFF", "BASICZBR3")
+    .device_automation_triggers(
+        {
+            # Power On action - uses identify command without parameters
+            (POWER_ON,): {
+                COMMAND: "identify",
+                CLUSTER_ID: 3,  # Identify cluster
```

Let's use the cluster objects here: `from zigpy.zcl.clusters.general import Identify` and then use `Identify.cluster_id`.

---

I attempted to use `Identify.cluster_id` as suggested, but the QuirkBuilder seems to convert it to `frozendict({})` instead of the integer value `3`. 

Could you clarify the correct way to reference cluster constants in v2 quirks? The tests pass with the hardcoded value but fail with the constant.

---

Actually, can you clarify where these commands are coming from? This automation trigger would match the device sending the *coordinator* an `identify` command. Do you have a debug log of it sending this command? I don't think I've seen that happen before.

---

Looking back at the original issue, I think I misunderstood the problem. The issue is showing device actions in the Home Assistant UI not automation triggers. Only now it's clear to me that this is probably a Home Assistant ZHA integration issue with how it presents device actions for the BASICZBR3, not a quirk issue. For clarification I used the default values for effect_variant in the matter specification (based on Zigbee). I apologize for the inconvenience. I will close the PR as not relevant.